src/coreclr/nativeaot/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -288,9 +288,12 @@ C_FUNC(\Name):
         movq    _\Var@TLVP(%rip), %rdi
         callq   *(%rdi)
 #else
-        leaq    \Var@TLSLD(%rip), %rdi
+        .byte 0x66  // data16 prefix - padding to have space for linker relaxations
+        leaq    \Var@TLSGD(%rip), %rdi
```

Comment:
LD form has no advantages over GD form here and GD end result would be slightly more compact, so I changed this to GD form, that also matches the code emitted for the inline TLS access.

Reply:
Just to add more details here. LD form has advantages when multiple variables are accesses from the same store in the same method. 
For the "TLS root" case we have just one variable, so the advantages would never be realized.